controllers/serviceaccount/serviceaccount_remover.go

```diff
@@ -0,0 +1,109 @@
+package serviceaccount
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudogu/cesapp/v4/core"
+	"github.com/cloudogu/cesapp/v4/registry"
+	k8sv1 "github.com/cloudogu/k8s-dogu-operator/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+// Remover is the unit to handle the removal of service accounts
+type Remover struct {
+	Registry registry.Registry `json:"registry"`
+	Executor CommandExecutor   `json:"executor"`
+}
+
+// NewServiceAccountRemover creates a new instance of ServiceAccountCreator
+func NewServiceAccountRemover(registry registry.Registry, commandExecutor CommandExecutor) *Remover {
+	return &Remover{
+		Registry: registry,
+		Executor: commandExecutor,
+	}
+}
+
+// RemoveServiceAccounts removes all service accounts for a given dogu
+func (r *Remover) RemoveServiceAccounts(ctx context.Context, doguResource *k8sv1.Dogu, dogu *core.Dogu) error {
+	logger := log.FromContext(ctx)
+
+	for _, serviceAccount := range dogu.ServiceAccounts {
+		parentPath := "sa-" + serviceAccount.Type
+		doguConfig := r.Registry.DoguConfig(dogu.GetSimpleName())
+
+		// check if the service account really exists
```

Comment:
There are a lot of redundant comments when the code expresses this perfectly fine?

To paraphrase the Clean Code idea about comments: 
> Comment only when the code cannot (sufficiently) express what is about to happen.